### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Therealadetunji**
I modified the Html and CSS to get the desktop view and also getting the mobile view if needed.

### CODE REVIEW

**Review by omar-labana — APPROVED** [review]
# **Invalid Code Review Request ⚠️❌**

Hi @ ,

While you made great effort in this project, unfortunately🚩, I cannot proceed to review your code😓.

The repo contains no changes regarding desktop version(media queries and other styling requirements), I believe you might be missing some commit since there is only the mobile version.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Portfolio.html [hunk 1]
```diff
@@ -5,321 +5,396 @@
...
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
-        <h3>LET’S CONNECT</h3>
+    <header class="toolbar">
+      <div class="logo">
+        <a id="LOGO" href="#"><b>My Logo</b></a>
+      </div>
+      <nav class="icon">
+        <img src="Images/Union.png" alt="Menu icon" />
```
Comment (koachcode) [inline Portfolio.html]: - Please check your images, none of them are coming up, I notice your path is `Images` with uppercase `I`, but your folder that contains all images is `images` with a lowercase `i`. Checking the folder name to `Images` with uppercase `i` could fix this for all image errors.

##### Yourr design
Screenshot from 2022-06-07 04-40-09

##### Figma Design
Screenshot from 2022-06-07 04-45-02

File: Portfolio.html [hunk 2]
```diff
@@ -5,321 +5,396 @@
...
-                <a href="#">
-                  <img src="./images/Github.png" alt="github icon" />
-                </a>
+            <ul>
+              <li>
+                <a href="#"
+                  ><img
+                    src="./Images/twitter.png"
+                    alt="Twitter Icon"
+                    class="twitter"
```
Comment (koachcode) [inline Portfolio.html]: - Please try to work on the about page to look exactly like what is the  Figma design. use the Figma background color and image for the specific section and other details from the Figma design should please be followed as well.
- Please work on your contact section as well to look like that of the Figma design, so as to keep with the detailed design. 

##### Your design
Screenshot from 2022-06-07 04-49-40
 
##### Figma design 
Screenshot from 2022-06-07 04-53-03

File: Portfolio.html [hunk 3]
```diff
@@ -5,321 +5,396 @@
...
+                <img src="Images/css-elipse.png" alt="Css Icon" />
                 <h3>CSS</h3>
               </li>
+              <li class="desktop-version">
+                <img src="./images/html.png" alt="html-tag"/>
+                <hx>HTML</hx>
+              </li>
+              <li class="lname">
+                <img src="Images/js-Ellipse.png" alt="Js Icon" />
+                <h3>Javascript</h3>
```
Comment (koachcode) [inline Portfolio.html]: - Please try to properly align-items on the webpage, as I noticed a breakage while resizing your webpage to a mobile view., and use the Figma design as a reference.

##### Figma design
Screenshot from 2022-06-07 04-59-57


##### Your design 
Screenshot from 2022-06-07 04-57-43

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @Therealadetunji , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linter errors :+1: 
- Good PR Message :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- You made use of `portfolio.html`, I would suggest its better to make use on `index.html`, as it would be easier to use that for the deployment of your website.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Therealadetunji 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**